projenrc/github-workflows.ts

```diff
@@ -573,7 +573,7 @@ export function buildCodeGenerationWorkflow(project: AwsCdkConstructLibrary) {
 
   // second job to create a PR if any changes were detected
   const pr: Job = {
-    name: 'Code generation',
+    name: 'Create Pull Request',
```

Comment:
Curious about the name change.

Reply:
Previously both jobs had the same name ```Code generation```. While this is accurate for the first job, the second one is focused on creating the PR with the changes detected